packages/xdl/src/Webpack/WebpackConfig.ts

```diff
@@ -0,0 +1,92 @@
+import { AnyConfiguration } from '@expo/webpack-config/webpack/types';
+import chalk from 'chalk';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+import { CLIWebOptions, WebEnvironment } from './Webpack.types';
+import { isDebugModeEnabled } from './WebpackEnv';
+
+export function setMode(mode: 'development' | 'production' | 'test' | 'none'): void {
```

Comment:
we support more "modes" here than we do for app.config.js https://github.com/expo/expo-cli/pull/1342/files#diff-bf738e4e35c755a9c5f9377a673b89e6R85